fastlane/assets/IMAGE_LICENSE.md

```diff
@@ -0,0 +1 @@
+Google open source projects may contain logos, trademarks or service names. These are "Brand Features", and if you want to use them, please do so consistent with [google.com/permissions/trademark/rules.html](https://www.google.com/permissions/trademark/rules.html).
```

Comment:
I worry legal will have issues with using the word LICENSE in the file name. What about IMAGE_GUIDELINES?